Ramda/Extensions/find.swift

```diff
@@ -20,7 +20,7 @@ extension R {
 
     */
 
-    public class func find<A, B: SequenceType where A == B.Generator.Element>(predicate: (A) -> Bool, in sequence: B) -> A? {
+    private class func find<A>(predicate: (A) -> Bool, in sequence: [A]) -> A? {
```

Comment:
Also not sure why we have to change it to an array, maybe you can show me because those images should fail because of the ambiguity. I'm probably missing something somewhere